R/plot.R

```diff
@@ -197,69 +320,69 @@ RadialSUCRA <- function(SUCRAData, ColourData, BUGSnetData, colourblind=FALSE) {
+    dat.edges$lwdA[comp.i+1] <- lwd.minA + (edges$edge.weight[i] - study_min) * (lwd_rangeA / study_range)
     comp.i <- comp.i + 2
     ID <- ID + 1
   }
   # add lines #
   CreateNetwork <- function(Type) {
-    if (Type=='Original') {
-      g <- ggplot(dat.edges, aes(x=reorder(treatment,-SUCRA), y=SUCRA, group=pairwiseID)) +
-        geom_line(linewidth=dat.edges$lwdO,show.legend = FALSE) +
-        scale_y_continuous(limits=c(-40,115))
+    if (Type == 'Original') {
+      g <- ggplot(dat.edges, aes(x = reorder(treatment, -SUCRA), y = SUCRA, group = pairwiseID)) +
+        geom_line(linewidth = dat.edges$lwdO, show.legend = FALSE) +
+        scale_y_continuous(limits = c(-40, 115))
     } else {
-      g <- ggplot(dat.edges, aes(x=reorder(treatment,-SUCRA), y=-20, group=pairwiseID)) +
-        geom_line(linewidth=dat.edges$lwdA,show.legend = FALSE) +
-        scale_y_continuous(limits=c(-80,115))
+      g <- ggplot(dat.edges, aes(x = reorder(treatment, -SUCRA), y = -20, group = pairwiseID)) +
+        geom_line(linewidth = dat.edges$lwdA, show.legend = FALSE) +
+        scale_y_continuous(limits = c(-80, 115))
     }
     g +
       ggiraphExtra::coord_radar() + 
       theme(panel.background = element_rect(fill = "transparent"), plot.background = element_rect(fill = "transparent", color = NA), 
             axis.title = element_blank(), axis.text.y = element_blank(), axis.ticks = element_blank(), 
             axis.line = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), aspect.ratio = 1, 
             axis.text.x = element_blank()) +
-      annotate("text",x = rep(0.5,7), y = c(-3,17,37,57,77,97,115), label = c("0","20","40","60","80","100","SUCRA (%)"), size=2.5, family="sans")
+      annotate("text", x = rep(0.5, 7), y = c(-3, 17, 37, 57, 77, 97, 115), label = c("0", "20", "40", "60", "80", "100", "SUCRA (%)"), size = 2.5, family = "sans")
   }
-  Network <- CreateNetwork(Type='Original')
-  ggsave(filename = 'NetworkO.png', device = 'png', bg = 'transparent', width=5, height=5)
+  Network <- CreateNetwork(Type = 'Original')
+  ggsave(filename = 'NetworkO.png', device = 'png', bg = 'transparent', width = 5, height = 5)
   
-  Network <- CreateNetwork(Type='Alternative')
-  ggsave(filename = "NetworkA.png", device = 'png', bg = 'transparent', width=5, height=5)
+  Network <- CreateNetwork(Type = 'Alternative')
+  ggsave(filename = "NetworkA.png", device = 'png', bg = 'transparent', width = 5, height = 5)
   
   
   # Plot of just points to go on the very top #
-  CreatePoints <- function(Type, colourblind=FALSE) {
-    if (Type=='Original') {
-      g <- ggplot(SUCRAData, aes(x=reorder(Treatment, -SUCRA), y=SUCRA, group=1)) +
-        geom_point(aes(fill=SUCRA, size=SizeO), size=SUCRAData$SizeO, shape=21,show.legend=FALSE) +
-        scale_y_continuous(limits=c(-40,115))
+  CreatePoints <- function(Type, colourblind = FALSE) {
```

Comment:
Should this inner function be factored out and documented?

Reply:
Ok, you can leave them where they are, but can you add roxygen comments for these functions though please?